packages/components/src/local/hex-grid/helpers/hex-named.ts

```diff
@@ -0,0 +1,14 @@
+import SergeHex from '../../mapping/types/serge-hex'
+import SergeGrid from '../../mapping/types/serge-grid'
+
+/** convert this point in cell coordinates to lat/long
+ * @param {string} name location in 2d Hex coordinate system
+ * @param {SergeGrid<SergeHex<{}>>} grid grid of cells
+ * @returns {SergeHex} centre represented by coords
+ */
+const hexNamed = (name: string, grid: SergeGrid<SergeHex<{}>>): SergeHex<{}> | undefined => {
+  return grid.find(cell => cell.name === name)
+
+}
```

Comment:
```suggestion
const hexNamed = (name: string, grid: SergeGrid<SergeHex<{}>>): SergeHex<{}> | undefined => 
    grid.find(cell => cell.name === name)
```

Reply:
oh, there was more to this than I realised. I've learned a new pattern.